### PR TITLE
docs: breaking 24.04.1 into standalone md

### PR DESCRIPTION
**Pull request by matthew-hagemann**
- Linked out to 24.04.1 OEM provisioning flow as a standalone file with both GIS configuration and EULA configuration contained in a single document.
- Added a ToC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@
 Please report any bugs related to Ubuntu Desktop Provision on [Launchpad](https://bugs.launchpad.net/ubuntu-desktop-provision).
 We use the GitHub issue tracker only for issues related to the development of Ubuntu Desktop Provision itself.
 
+## 24.04.1 OEM provisioning
+
+For configuring an OEM provisioning flow for Ubuntu 24.04.1 LTS, where user creation and account setup is handled
+seperately to system setup, please refer to [this guide](oem-provisioning-24_04_1.md).
```
Comment (spydon) [inline README.md:15]: Should we maybe have all the docs in a docs directory, I'm guessing there will be more of them coming?

File: oem-provisioning-24_04_1.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
-# Why Gnome-Initial-Setup
+# 24.04.1 OEM provisioning flow
+
+To help support installation flows where user account setup is a seperate flow from system installation, we have 
+included this guide on how to invoke Gnome Initial Setup as part of the installation flow, either via an `autoinstall.yaml` or a
```
Comment (spydon) [inline oem-provisioning-24_04_1.md:4]: Line-length, I'll add a markdown linter in here at some point soon hopefully :smile:

**Review by spydon — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (matthew-hagemann) [reply README.md:15]: Yea, we are headed that way. Other than this doc for now, do you think we should also move the CONTRIBUTING.md in there or is it more idiomatic to keep next to the README?

**Review by d-loose — APPROVED** [review]
Thanks!

+1 for adding a `docs/` directory. According to the [github docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) the `CONTRIBUTING.md` can be moved there too

**Comment by matthew-hagemann** [comment]
@d-loose @spydon added `docs/` :+1:

**Pull request merged**